README.md

```diff
@@ -1,13 +1,38 @@
 # DataJoint Element - Functional Calcium Imaging
 
-DataJoint Element for functional calcium imaging with `ScanImage`, `Scanbox`,
-`Nikon NIS`, or `PrairieView` acquisition software and `Suite2p`, `CaImAn`, and
-`EXTRACT` analysis software. DataJoint Elements collectively standardize and automate
+DataJoint Element for functional calcium imaging with 
+[ScanImage](https://docs.scanimage.org/), 
+[Scanbox](https://scanbox.org/),
+[Nikon NIS-Elements](https://www.microscope.healthcare.nikon.com/products/software/nis-elements), 
+and `Bruker Prairie View` acquisition software; and 
```

Comment:
```suggestion
and `Bruker PrairieView` acquisition software; and 
```

Reply:
Thanks @kushalbakshi.  There might be a more accurate source, but the Bruker website lists it as `Prairie View`:

<img width="1313" alt="image" src="https://github.com/datajoint/element-calcium-imaging/assets/871137/56e719e3-d7c6-49a9-802a-f6988b6a1523">

Reply:
You're correct! Sorry about that. I thought it was one word.